config/action.d/pf.conf

```diff
@@ -75,14 +75,13 @@ protocol = tcp
 # Option:  port
 # Notes.:  specifies port to monitor
 # Values:  [ NUM | STRING ]  Default:
-#
-#port = telnet
+port = ssh
```

Comment:
I don't like such defauts values:
I think better it fails, as silently bans the IP for wrong port :)))
If used multiport, the port SHOULD be defined in jail, imho.